portfolio/src/main/webapp/index.html

```diff
@@ -53,14 +54,32 @@
                 <li>HTML</li>
                 <li>CSS</li>
             </ul>
-            <!--Adding a servlet to portofolio-->
-            <p>
-                Click <a href="/String">here</a> to access servlet
-            </p>
                 <button onclick="showFact();">Get fun fact</button>
-            <p id="fact-container">
-            </p>
+             <p id="fact-container">
+             </p>
+        </div>
+    </div>
+
+    <section id="contact">
+        <header><h1>Contact</h1></header>
+    <div class="contact-form" id="contact">
+        <div class="contact-content">
+            <label>Name: </label>
+            <input type="text" name="name" placeholder="Enter Your Name">
+
+            <label>Email: </label>
+            <input type="text" name="email" placeholder="Enter Your Email">
+
+            <label>Message: </label>
+            <form method="POST" action="/new-task">
+            <input type="text" name="title" />
+             <br/>
+            <a class="btn"><button>Submit</button></a>
+            </form>
```

Comment:
indentation

Reply:
I don't see any changes to the indentation in this section? The elements within the `form` should be further indented than the `<form>` tags